app-tracking-protection/app-tracking-api/src/main/java/com/duckduckgo/mobile/android/app/tracking/AppTrackingProtection.kt

```diff
@@ -47,4 +47,12 @@ interface AppTrackingProtection {
      * This method will stop the App Tracking Protection feature
      */
     fun stop()
+
+    /**
+     * This is a suspend function because the operation can take time.
+     * You DO NOT need to set any dispatcher to call this suspend function
+     *
+     * @return `true` when the app is excluded from App Tracking Protection
+     */
+    suspend fun isAppExcluded(packageName: String): Boolean
```

Comment:
nit: I was more thinking about getting the exclusion list as API, I think it's more generic. But fine

Reply:
I’ll address this in the base PR.